src/Symfony/Component/Console/Application.php

```diff
@@ -1033,8 +1033,7 @@ protected function getDefaultInputDefinition()
             new InputOption('--quiet', '-q', InputOption::VALUE_NONE, 'Do not output any message'),
             new InputOption('--verbose', '-v|vv|vvv', InputOption::VALUE_NONE, 'Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug'),
             new InputOption('--version', '-V', InputOption::VALUE_NONE, 'Display this application version'),
-            new InputOption('--ansi', '', InputOption::VALUE_NONE, 'Force ANSI output'),
-            new InputOption('--no-ansi', '', InputOption::VALUE_NONE, 'Disable ANSI output'),
+            new InputOption('--ansi', '', InputOption::VALUE_NEGATABLE, 'Force [or Disable] ANSI output', null),
```

Comment:
What about `Force (or disable --no-ansi) ANSI output`.
Or maybe we should automatically add something when an option is negatable: `Force ANSI output (can be disabled via --no-ansi)` where the `(can be disabled via --no-ansi)` part is automatically added by Symfony?

Reply:
I would love that kind of magic here 👍🏻

Reply:
mhhh.. what's about people that use a non-english description ?

I wouldn't use magic here and let people write the full sentences.

Reply:
I changed the sentence to `(or disable --no-ansi)`.

But I'm really not conviced by automated sentence:
- won't work for non-english application
- wording may vary with context (does disable (or whatever choosed word) will always work for this option?
- it's easy for the user to enter the right sentence.

Reply:
I agree 👍🏻